CondFormats/SiStripObjects/src/SiStripSummary.cc

```diff
@@ -62,9 +62,9 @@ const SiStripSummary::Range SiStripSummary::getRange(const uint32_t& DetId) cons
   RegistryIterator p = std::lower_bound(indexes_.begin(), indexes_.end(), DetId, SiStripSummary::StrictWeakOrdering());
   if (p == indexes_.end() || p->detid != DetId) {
     return SiStripSummary::Range(v_sum_.end(), v_sum_.end());
-    std::cout << "not in range " << std::endl;
-  } else
-    return SiStripSummary::Range(v_sum_.begin() + p->ibegin, v_sum_.begin() + p->ibegin + userDBContent_.size());
+    //  std::cout << "not in range " << std::endl;
```

Comment:
Also here, can we keep the printout (moving it above the return and making it a `LogWarning`?

Reply:
see my comment https://github.com/cms-sw/cmssw/pull/43030#discussion_r1361606548